### DISCUSSION TITLE
Implementing emojis (smileys)

### DISCUSSION THREAD

**Discussion by zorandjukic** [voidlabs/mosaico#629] [Ideas]
Hi there,

do you have in plan to add emojis (smileys) in Mosaico email builder?

Or do you have suggestion which library we could implement into Mosaico?

**Comment by bago** [comment]
We have no plan implementing emojis.

We'd like to move away from Tinymce as it is a PITA to use it in inline mode and it is full of bugs that are being ignored by tinymce developers (I guess their paying customers don't use inline mode).

You can try backporting the emoji tinymce plugin to tinymce 4.5.8 or try using the emoticons plugin for 4.x:
https://www.tiny.cloud/docs-4x/plugins/emoticons/

Have a look to #310 to see how to configure tinymce while loading mosaico.

**Comment by bago** [comment]
For the record upgrading Tinymce should be enabled since #644
